aspnetcore/fundamentals/request-response.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Request and Response operations in ASP.NET Core
+author: jkotalik
+description: Learn how to get started with WebSockets in ASP.NET Core.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: tdykstra
+ms.custom: mvc
+ms.date: 02/26/2019
+uid: fundamentals/request-response
+---
+# Request and response operations in ASP.NET Core
```

Comment:
Middleware is a common scenario, but this can apply to anything that reads/writes to/from the request/response.

The  more I think about it though, I think this should be in the middleware section though. Most of the time, if you are accessing HttpContext.Request.Body, you are in a middleware.